tsconfig.json

```diff
@@ -4,7 +4,8 @@
 		"target": "es6",
 		"outDir": "out",
 		"lib": [
-			"es6"
```

Comment:
Why are you changing this?

Reply:
I had to, to be able to compile on my computer (ubuntu 20). If its not necessary for you, just skip it.

Reply:
I'll check on my machine before publishing. If it works I don't mind.